File: segundo-semestre/lab121/java/cuatro.java
import java.util.Scanner;

class Cuatro {
  public static void main(String args[]) {
    Scanner reader = new Scanner(System.in);
    int x;
    int y;

    System.out.println("Dame 2 numeros: ");
    x = reader.nextInt();
    y = reader.nextInt();

    if (x != y) {
      if (x > y) {
        System.out.println("x es mayor");
      } else {
        System.out.println("y es mayor");
      }
    }
    System.out.println("son iguales");
  }
}

File: segundo-semestre/lab121/tarea1/c_sharp/cuatro.cs
class MainClass {
  public static void Main(string[] args) {
    int x;
    int y;

    System.Console.WriteLine("Dame dos numeros: ");

    x = System.Convert.ToInt32(System.Console.ReadLine());
    y = System.Convert.ToInt32(System.Console.ReadLine());

    if (x != y) {
      if (x > y) {
        System.Console.WriteLine("x es mayor");
      } else {
        System.Console.WriteLine("y es mayor");
      }
    }

    System.Console.WriteLine("son iguales");
  }
}
